src/util.rs

```diff
@@ -506,6 +506,14 @@ pub fn daemonize() -> Result<()> {
     Ok(())
 }
 
+/// Disable connection pool to avoid broken connection between runtime
```

Comment:
nit: I'd prefer a reference to the bug this caused or a more elaborate explanation inline

Reply:
Got it, fixed!